docs/api/intro.md

```diff
@@ -6,22 +6,15 @@ sidebar_label: Intro
 
 ## Exports
 
-CARMI exports only a few APIs -
+CARMI exports only several APIs:
 
-1.  compile(model, {compiler,name,output}) - takes a model (a plain JS object with two types of properties - either a
-    state derivations or a setter) and generates the source code for a function which receives two parameters - the
-    actual initial state and an optional function library and returns an object with the defined derivations and
-    setters. Whenever you trigger a setter the derivations update incrementally
-2.  root - a token which represents the root of your state, which is used to create your derived state Directed Acyclic
```

Comment:
I removed the "Directed Acyclic Graph," section, since I am not sure what the comma there meant - either this is an incomplete sentence, or this part shouldn't be there.